src/daemon/db_util.rs

```diff
@@ -64,7 +64,7 @@ fn load_all_forest_cars_internal<T>(
         .into_iter()
         .filter_map(|e| {
             e.ok().and_then(|e| {
-                if e.file_type().is_file() {
```

Comment:
Tests whether this file type represents a regular file. The result is mutually exclusive to the results of [is_dir] and [is_symlink]; only zero or one of these tests may pass.